database/test-schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "users": {
+    "user_id_1": {
+      "brooklyn_completed_tasks": "",
+      "email": "taylor.swift@scl.com",
+      "manhattan_completed_tasks": {
+        "id_13": true
```

Comment:
I'm envisioning these `id_X` as 5 digit alphanumerics, I think that should be ok, as long as they don't start with numbers? i.e. `u34xn`

Reply:
5 digits alphanumeric? i think they're all treated as strings no matter what with the rules of JSON, so probably supports anything that is a valid key (0-9,A-z). This doesnt matter here except symbols. 

I think it would be cool to embed the tasks and id's instead of just a "true" since we do a presence methodology but we could also include them all and do something like

```
"id_01": false
"id_02": true
"id_03": true
"id_05": false 
```

what do you think about using more and loading more data and enriching the user context even if its junk data

Reply:
could be one of those things payload counting depending on the max size then but it shouldn't except like a few kbs if not like 100kbs

Reply:
As for the id's, AirTable autogenerates some that we can use helpfully. I'm not against presetting them as true/false, but I think we just don't know at app load time, so keeping them only to `true` I admit is not the most elegant, but is easier for now